recipes/emboss/fix_acd_path.py

```diff
@@ -14,7 +14,7 @@
 export EMBOSS_DATA=../share/EMBOSS/data
 export PLPLOT_LIB=../share/EMBOSS
 
-_water $@
+_water "$@"
```

Comment:
Maybe single quotes? Do we want the shell code to be evaluated?

Reply:
We do want to allow environment variables to be used, e.g.

```
$ water -outfile="$PWD/temp with space.water" -asequence=asis:ACCCGGGCGCGGT -bsequence=asis:ACCCGAGCGCGGT -gapopen=10 -gapextend=0.5
```

Of course, ``$PWD`` is a silly example - something like ``$PROJECT`` or whatever might make sense.

Anyway, this example breaks using  ``_water '$@'`` so the double quote version seems better to me.

Reply:
Makes sense, just double checking.